docs/_docs/code-snippets/k8s/stateless/deployment.yaml

```diff
@@ -0,0 +1,85 @@
+# An example of a Kubernetes configuration for pod deployment.
```

Comment:
We should use already existing template file instead of creating random new one.